packages/moonstone/Scrollable/ScrollButtons.js

```diff
@@ -252,6 +260,33 @@ class ScrollButtons extends Component {
 		}
 	}
 
+	focusOnOppositeScrollButton = (ev, direction) => {
+		const buttonNode = (ev.target === this.nextButtonNodeRef) ? this.prevButtonNodeRef : this.nextButtonNodeRef;
+
+		ev.preventDefault();
+		ev.nativeEvent.stopPropagation();
+
+		if (!Spotlight.focus(buttonNode)) {
+			Spotlight.move(direction);
+		}
+	}
+
+	handleSpotlight = (ev) => {
+		const
+			{rtl, vertical} = this.props,
+			{prevButtonDisabled, nextButtonDisabled} = this.state,
+			{keyCode, target} = ev,
+			direction = getDirection(keyCode),
+			fromNextToPrev = (vertical && direction === 'up') || (!vertical && direction === (rtl ? 'right' : 'left')),
+			fromPrevToNext = (vertical && direction === 'down') || (!vertical && direction === (rtl ? 'left' : 'right'));
+
+		// manually focus the opposite scroll button when 5way pressed
+		if ((fromNextToPrev && target === this.nextButtonNodeRef && !prevButtonDisabled) ||
```

Comment:
Do we need to do all these calculations?  It seems a bit complicated.  Should we just allow spotlight to fail to focus the opposite button when it's disabled?  Is there any other way to simplify this?

Reply:
I guess we could remove the condition of seeing disabled but I'm afraid we cannot remove the rest.
`focusOnOppositeScrollButton` should be only called when the target is button node and it should be matched with the desired direction otherwise, it behaves wrong. If we remove those conditions, pressing 'left' key from 'up' button, it goes to 'down' button which is we don't want to.